### PR TITLE
fix team owner style

### PR DESCRIPTION
**Pull request by laushinka**
## Description
Quick style fix for Project Search team owner property.

|BEFORE|AFTER|
| - | - |
| Screenshot 2022-02-03 at 10 34 52 | Screenshot 2022-02-03 at 10 34 21 |

## Related Issue(s)
no-issue

## How to test
1. Add a project to a team.
2. Go to https://gitpod-staging.com/admin/projects and search for the project. The detail should show the team owner style as in the description.

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **laushinka** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by laushinka** [comment]
@gtsiolis This is just a quick fix because I forgot to also adapt the style for the team owner property.

**Review by JanKoehnlein — APPROVED** [review]
LGTM

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7999#pullrequestreview-871663021" title="Approved">JanKoehnlein</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [JanKoehnlein]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: f91475884d138e33d6ad23356a5f317168a50933</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7999#pullrequestreview-871663021" title="Approved">JanKoehnlein</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/7999#issuecomment-1028874467" title="Approved">laushinka</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/7999#issuecomment-1028874467" title="Approved">laushinka</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [JanKoehnlein,laushinka]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by gtsiolis** [comment]
Thanks @laushinka @JanKoehnlein! 🏀 

**issue**: One more edge case affecting this page is when the project has been soft deleted and there's no team association and the team is removed from the owner section.

**suggestion**: FWIW, what we could do is render a dash for indicating the missing team or user in front of the owner type, but it should be fine to leave this as is of now as more improvements landing in the admin dashboard could make this issue obsolete.

**Comment by laushinka** [comment]
@gtsiolis Ah, good point. Yess, look forward to making more improvements 👏🏼

**Pull request merged**
